var/spack/repos/builtin/packages/bigdft-atlab/package.py

```diff
@@ -39,20 +42,31 @@ def configure_args(self):
         prefix = self.prefix
 
         fcflags = []
+        cflags = []
+        cxxflags = []
+
         if "+openmp" in spec:
             fcflags.append(self.compiler.openmp_flag)
 
+        if "+shared" in spec:
```

Comment:
This is a pretty common variant so you probably want to use `spec.satisfies()` here and other places where you are checking that the package's spec has the variant setting.

See docs (https://spack.readthedocs.io/en/latest/packaging_guide.html#testing-spec-constraints):

"The difference between the two functions is that satisfies() tests whether spec constraints overlap at all, while in tests whether a spec or any of its dependencies satisfy the provided spec."

Reply:
done in  425e8de